store/tikv/kv.go

```diff
@@ -128,8 +136,10 @@ func NewKVStore(uuid string, pdClient pd.Client, spkv SafePointKV, client Client
 		replicaReadSeed: rand.Uint32(),
 	}
 	store.lockResolver = newLockResolver(store)
+	store.safeTSMu.storeSafeTS = make(map[uint64]uint64)
```

Comment:
Can we reduce the concept of `xxxxts`? There are already have `start ts`, `commit ts`, `commit ts`, `read ts`, `resolve ts` and so on, maybe directly name it as `resolve ts`.

Reply:
updated.